srcpkgs/ft2-clone/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'ft2-clone'
+pkgname=ft2-clone
+version=1.21
```

Comment:
1.25 is out

Reply:
Changed version, commit name and PR name to reflect version